src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -1,16 +1,45 @@
 package seedu.address.model.util;
 
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.label.Label;
+import seedu.address.model.tag.FileAddress;
 import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.TagName;
+
 
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
     public static Tag[] getSampleTags() {
-        return new Tag[] {};
```

Comment:
This part should not be changed. This are the default tags when HelloFile is launched the first time. We might want to keep this list empty?

Reply:
Okay amend it!